modules/MwEmbedSupport/mediawiki/mediawiki.client.js

```diff
@@ -88,7 +88,7 @@
 		return (mw.isAndroid() && !mw.isFirefox() && !mw.isChrome());
 	};
 	mw.isAndroidChromeNativeBrowser = function () {
-		return ( mw.isAndroid() && mw.isChrome() && userAgent.indexOf('Version/') != -1 )
+		return ( mw.isAndroid() && mw.isChrome() );
```

Comment:
@InbarItay  - this change now returns true for regular Chrome on Android. Please verify.